File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.java
class Solution {
    public int minimumLength(String s) {
        HashMap<Character,Integer> freq = new HashMap<>();
        for(char ch : s.toCharArray()){
            freq.put(ch,freq.getOrDefault(ch,0)+1);
        }

        int delCnt = 0;
        for(int f: freq.values()){
            if(f%2==1)delCnt += (f-1);
            else delCnt +=(f-2);
        }
        return (s.length()-delCnt);
        
    }
}

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int> freq(26,0);
        int ans(0);
        for(auto ch : s)freq[ch-'a']++ ; 

        for(auto f : freq){
            if(f==0)continue;
            else if(f%2==0)ans+=2;
            else ans+=1;
        }
        return ans;
    }
};